docs/en/datasets/det.md

```diff
@@ -212,3 +218,32 @@ rm dataset.zip && rm -rf dataset
 ```bash
 python tools/data/textdet/funsd_converter.py PATH/TO/funsd --nproc 4
 ```
+
+### ILST
+- Step1: Download dataset from [here](https://iiitaphyd-my.sharepoint.com/:f:/g/personal/minesh_mathew_research_iiit_ac_in/EtLvCozBgaBIoqglF4M-lHABMgNcCDW9rJYKKWpeSQEElQ?e=zToXZP)
+- Step2: Run the following commands
+```bash
+unzip -q IIIT-ILST.zip && rm IIIT-ILST.zip
+cd IIIT-ILST
+# rename files
+cd Devanagari && for i in `ls`; do mv -f $i `echo "devanagari_"$i`; done && cd ..
+cd Malayalam && for i in `ls`; do mv -f $i `echo "malayalam_"$i`; done && cd ..
+cd Telugu && for i in `ls`; do mv -f $i `echo "telugu_"$i`; done && cd ..
+# transfer image path
+mkdir imgs && mkdir annotations
+mv Malayalam/{*jpg,*jpeg} imgs/ && mv Malayalam/*xml annotations/
+mv Devanagari/*jpg imgs/ && mv Devanagari/*xml annotations/
+mv Telugu/*jpeg imgs/ && mv Telugu/*xml annotations/
+# unify postfix
+cd imgs
+ls -1 *.jpeg | xargs -n 1 bash -c 'convert "$0" "${0%.jpeg}.jpg"'
+rm -f *jpeg
```

Comment:
why do we need to do this?

Reply:
There are .jpg images and .jpeg images in the same file. So I use command `line239` to rename all the .jpeg  as .jpg. However, this operation will make a copy of the original `jpeg` images. So another remove command is used.

Reply:
> There are .jpg images and .jpeg images in the same file. So I use command `line239` to rename all the .jpeg as .jpg. However, this operation will make a copy of the original `jpeg` images. So another remove command is used.

Yes, I know. I mean why do we need to rename all jpeg to jpg?